app/misc/graphing.ts

```diff
@@ -122,6 +122,7 @@ function populateCommits(oldResult) {
         commitList = [];
         parentCount = {};
         columns = [];
+        branchIds = [];
```

Comment:
Solves #29. Previously this was not cleared out so upon refreshing the graph it would already have existing values. The white node was a pre-existing branch in a previous revision of the graph (before commiting).